cli/repo/repo_update.go

```diff
@@ -87,8 +92,32 @@ func repoUpdate(ctx context.Context, c *cli.Command) error {
 	if c.IsSet("trusted") {
 		patch.IsTrusted = &trusted
 	}
+	// TODO: remove isGated in next major release
 	if c.IsSet("gated") {
-		patch.IsGated = &gated
+		if gated {
+			allEvents := "all_events"
+			patch.RequireApproval = &allEvents
+		} else {
+			forks := "forks"
+			patch.RequireApproval = &forks
+		}
+	}
+	if c.IsSet("require-approval") {
+		switch requireApproval {
+		case "forks", "pull_requests", "all_events":
```

Comment:
Can we use constants for these strings?

Reply:
Ive seen there is already `RequireApprovalAllEvents` etc. can we not use it here as well?

Reply:
I just copied the types from server/models to woodpecker-go now.